source/includes/examples/_cart-pole.html.md

```diff
@@ -1,14 +1,12 @@
-## Example: Cart Pole
+# Cart Pole Example
```

Comment:
I think it would make more sense to leave them as it because when you do a search or a filter on the docs website they would come up as just "Cart Pole" and yeah, I don't think people know what that is.